package.json

```diff
@@ -4,8 +4,17 @@
   "version": "3.8.1",
   "license": "Apache-2.0",
   "description": "Angular extension of the PatternFly project.",
+  "keywords": ["angular", "patternfly"],
+  "main": "index.js",
   "homepage": "https://github.com/patternfly/angular-patternfly",
-  "dependencies": {},
+  "dependencies": {
+    "angular": "1.3.0 - 1.5.*",
+    "angular-animate": "1.3.0 - 1.5.*",
+    "angular-sanitize": "1.3.0 - 1.5.*",
+    "angular-ui-bootstrap": "0.13.x",
+    "lodash": "3.x",
```

Comment:
It appears Lodash is still in use. I see references to `._remove`, `_.findWhere` and `_.find` which are not scoped like the references to `_.merge` which check for `angular.merge` first.

I think we should address this in a separate PR.